docker/docker.go

```diff
@@ -25,6 +26,14 @@ const (
 )
 
 func main() {
+
+	for j := 1; j < len(os.Args); j++ {
+		if os.Args[j] == "-d" || os.Args[j] == "--daemon" {
+			log.Println("-d is deprecated. Please use daemon command")
+			os.Args[j] = "daemon"
+		}
+	}
+
```

Comment:
This isn't backwards compatible. Consider `sudo docker -g ./graph -p ./d.pid -d`.